src/server/pachyderm_test.go

```diff
@@ -3590,9 +3590,19 @@ func scalePachd(t testing.TB) {
 }
 
 func getKubeClient(t testing.TB) *kube.Client {
-	config := &kube_client.Config{
-		Host:     "http://0.0.0.0:8080",
-		Insecure: false,
+	var config *kube_client.Config
+	host := os.Getenv("KUBERNETES_SERVICE_HOST")
+	port := os.Getenv("KUBERNETES_SERVICE_PORT")
```

Comment:
Have you look into this?

https://github.com/kubernetes/client-go#how-to-use-it